src/openforms/conf/base.py

```diff
@@ -1165,15 +1161,27 @@
 #
 # Django-log-outgoing-requests
 #
+LOG_OUTGOING_REQUESTS_DB_SAVE = config("LOG_OUTGOING_REQUESTS_DB_SAVE", default=False)
+LOG_OUTGOING_REQUESTS_DB_SAVE_BODY = config(
+    "LOG_OUTGOING_REQUESTS_DB_SAVE_BODY", default=False
+)
 LOG_OUTGOING_REQUESTS_CONTENT_TYPES = [
     ContentType(pattern="application/json", default_encoding="utf-8"),
     ContentType(pattern="application/soap+xml", default_encoding="utf-8"),
     ContentType(pattern="application/xml", default_encoding="utf-8"),
     ContentType(pattern="text/xml", default_encoding="iso-8859-1"),
     ContentType(pattern="text/*", default_encoding="utf-8"),
 ]
-LOG_OUTGOING_REQUESTS_EMIT_BODY = True
+LOG_OUTGOING_REQUESTS_EMIT_BODY = config(
+    "LOG_OUTGOING_REQUESTS_EMIT_BODY", default=True
+)
 LOG_OUTGOING_REQUESTS_MAX_CONTENT_LENGTH = 524_288  # 0.5MB
+# number of days
+LOG_OUTGOING_REQUESTS_MAX_AGE = config("LOG_OUTGOING_REQUESTS_MAX_AGE", default=7)
+# number of minutes, set to 0 to disable reset
+LOG_OUTGOING_REQUESTS_RESET_DB_SAVE_AFTER = config(
+    "LOG_OUTGOING_REQUESTS_RESET_DB_SAVE_AFTER", default=60
+)
```

Comment:
Do I understand it correctly that having `LOG_OUTGOING_REQUESTS_DB_SAVE=True` makes this setting obsolete?

Reply:
not really - you can disable saving in the admin, and then after 60 mins (by default), it will be reset to the "settings default", which in your example would re-enable it. It's a bit of a weird flow - nobody is likely to do this, but it does highlight the implementation details.